### ISSUE TITLE
HyperEdit's Debug menu button is brokey.

### ISSUE THREAD

**Issue by Ezriilc** [Ezriilc/HyperEdit#38]

**Comment by fronbow** [comment]
Yes, this was one thing that got me looking at the source originally!
When I understand the View side of things it should be easier to fix!

**Comment by fronbow** [comment]
For the moment (latest test build) I've removed the menu option. Until such time as I can work out how to get it to show up again!
